lib/roo_on_rails/checks/heroku/app_exists.rb

```diff
@@ -16,6 +16,10 @@ module Heroku
       # Output context:
       # - heroku.app.{env}: an app name.
       class AppExists < EnvSpecific
+        ACCEPTABLE_ENV_NAMES = {
+          'production' => ['prod', 'production'],
+          'staging'    => ['stg', 'staging']
```

Comment:
Having done a little bit of research, there isn't much consensus around environment name abbreviations. We largely use `staging` so I think `test` would be a significant shift, and I have an emotional reaction to `stag` because said out loud it doesn't make me think of an environment 🦌 .  One source suggested `prd` and `stg` which I think aligns nicely with your point about character length @humzashah. 

My understanding of `roo_on_rails` is that it configures / validates each environment separately, so it'd be difficult to check that both environment use the same style of abbreviations, but I'm not very well versed in the code. @humzashah  @mezis  do you have any suggestions around how we could accomplish that?